src/components/forms/CreateSmsTemplate/form-actions.ts

```diff
@@ -0,0 +1,7 @@
+export const calculateHowManySmsMessages = (val: number) => {
```

Comment:
strange location and file name for this

Reply:
I did rename this to view-actions.ts in another PR. 

I broke this out into it's own file to reduce the noise in the view page. I'll rename the file here.

Reply:
I've rename this file to be more appropriate and added a comment on the function. 

The other reason I wanted to move this into a different file, is that I wanted to test this in isolation.